tests/integration/profile_test.py

```diff
@@ -86,6 +102,16 @@ def test_delete_existing_interface_inactive_profiles(eth1_up):
         assert not profile_exists
 
 
+def test_rename_existing_interface_active_profile(eth1_up):
+    with create_active_profile(eth1_up[Interface.KEY][0][Interface.NAME]):
+        eth1_up[Interface.KEY][0][Interface.MTU] = 2000
+        libnmstate.apply(eth1_up)
+        profile_exists = _profile_exists(
+            ETH_PROFILE_DIRECTORY + "ifcfg-testProfile"
+        )
+        assert profile_exists
```

Comment:
nit: I think you can assert directly the `_profile_exists` result.

Reply:
Done, thanks!